other/materials_designer/create_box_cutout.ipynb

```diff
@@ -0,0 +1,283 @@
+{
```

Comment:
<p>Can we add an image to demonstrate what MAX, MIN_COORDINATES are</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/163/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>